stable/kube-ops-view/Chart.yaml

```diff
@@ -1,5 +1,6 @@
+apiVersion: v1
 name: kube-ops-view
-version: 0.8.0
+version: 1.0.0
```

Comment:
Shouldn't this be `0.8.1`?

Reply:
we agreed some time ago that all charts in the stable folder should be at least 1.0.0 @mattfarina